email_automation/email-config.py

```diff
@@ -0,0 +1,171 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+import os
+import logging
+import sys
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+#Input the filename as an argument in command line 
+filename_raw = sys.argv[1]
+filename = filename_raw.split(".")[0]
+pool_name = "nrelopenpath-prod-" + filename
+
+current_path = os.path.dirname(__file__)
+config_path = os.path.relpath('../configs/'+ filename_raw, current_path)
+
+#Set up AWS credentials as environment variables + set variables 
+ACCESS = os.environ.get("AWS_ACCESS_KEY_ID")
+SECRET = os.environ.get("AWS_SECRET_ACCESS_KEY")
+TOKEN = os.environ.get("AWS_SESSION_TOKEN")
+AWS_REGION = "us-west-2"
+
+#Set up clients
+cognito_client = boto3.client(
+    'cognito-idp',
+    aws_access_key_id = ACCESS,
+    aws_secret_access_key= SECRET,
+    aws_session_token=TOKEN, 
+    region_name=AWS_REGION
+    )
+sts_client = boto3.client("sts")
+
+
+def get_userpool_name(pool_name, cognito_client):
+    response = cognito_client.list_user_pools(MaxResults=60)
+    UserPoolExist = False
+    #########One option to set the user pool without breaking (but still stop when condition met)
+    i = 0
+    while response["UserPools"][i]["Name"] != pool_name and i < len(response["UserPools"]) - 1:
+        print("looking for user pool...")
+        i = i + 1
+        if response["UserPools"][i]["Name"] == pool_name:
+            UserPoolExist = True
```

Comment:
nit: you generally don't want to have variable names start with a capital - that looks like a class name. `is_userpool_exist` may be better.

Reply:
renamed exactly as you suggested!